baselines/kms/proto.list.baseline

```diff
@@ -3,7 +3,7 @@ google/protobuf/descriptor.proto
 google/api/annotations.proto
 google/protobuf/duration.proto
 google/protobuf/timestamp.proto
-google/kms/v1/resources.proto
+google/cloud/kms/v1/resources.proto
```

Comment:
I'm a little bit confused here, if we generated `google/kms/v1/resources.proto` before instead of `/google/cloud/kms....` in proto_list, that sounds like a bug?

Reply:
Originally `kms` lived in `protos/google/kms`, I just moved it to `protos/google/cloud/kms` to match the googleapis location.